package/MDAnalysis/analysis/hydrogenbonds/wbridge_analysis.py

```diff
@@ -866,8 +865,9 @@ def __init__(self, universe, selection1='protein',
             :attr:`~DEFAULT_ACCEPTORS` values.
             ["CHARMM27"]
         donors : sequence (optional)
-            Extra H donor atom types (in addition to those in
-            :attr:`~DEFAULT_DONORS`), must be a sequence.
+            Extra H donor atom types (in addition to those in :attr:`~DEFAULT_DONORS`).
+            This shall be the name of the heavy atom that bonded to the hydrogen.
+            For example, the oxygen ('O') in the hydroxyl group, must be a sequence.
```

Comment:
```suggestion
            For example, the oxygen ('O') in the hydroxyl group. Must be a sequence.
```
My understanding of that sentence is that the intent was for a full stop?